datalad/distribution/dataset.py

```diff
@@ -287,17 +290,25 @@ def get_subdatasets(self, pattern=None, fulfilled=None, absolute=False,
             for sm in submodules:
                 sdspath = opj(self._path, sm)
                 rsm.extend(
-                    [opj(sm, sdsh)
+                    [(normpath(opj(sm, sdsh[0])), opj(sm, sdsh[1])) if isinstance(sdsh, tuple) else opj(sm, sdsh)
```

Comment:
is normpath use safe here?  generally, we managed to avoid using it so we could live happily with symlinked sub-datasets etc